### ISSUE TITLE
Consider a plugin model

### ISSUE THREAD

**Issue by jmarnold** [FubuMvcArchive/ripple#102]
It's come up a few times already. Maybe pull from a directory inside of the ripple directory for DLLs? Probably useful for two things:
1. Visiting all RippleOperations
2. Providing custom commands that leverage the ripple API

**Comment by SimonCropp** [comment]
-1  if people want to reuse ripple functionality let them reference the exe and use it like a library

**Comment by jmarnold** [comment]
I agree. Just had this as a placeholder in case folks REALLy wanted it.

**Comment by chester89** [comment]
I would like that very much.
